lib/src/cache/in_memory.dart

```diff
@@ -2,11 +2,25 @@ import 'dart:async';
 import 'dart:collection';
 import 'dart:convert';
 import 'dart:io';
-
 import 'package:path_provider/path_provider.dart';
-
```

Comment:
We also tend to keep project packages separate from dart and installed packages.